### DISCUSSION TITLE
Grafana 8.0

### DISCUSSION THREAD

**Discussion by pedropombeiro** [marcusolsson/grafana-hourly-heatmap-panel#32] [Q&A] [answered]
Hi, I noticed that the panel doesn't seem to be working on Grafana 8.0 after migration from 7.0:

> Cannot read property 'visualization' of undefined

Are there plans to migrate it to 8.0?

**Comment by marcusolsson** [comment, marked as answer]
No migration needed. I've added the fix for this today so I'll publish this tomorrow.

**Reply by pedropombeiro** [reply]
Thanks, just restarted my Grafana instance and it worked straight away!
